src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/AbstractDSLStrategy.java

```diff
@@ -35,6 +37,30 @@ public AbstractDSLStrategy(int tabs, IDescriptor descriptor) {
 	public AbstractDSLStrategy(int tabs, IDescriptor descriptor, boolean shouldInitChildren) {
 		this.tabs = tabs;
 		this.propertyDescriptor = descriptor;
+		propertiesToBeSkipped.add("actions");
+		propertiesToBeSkipped.add("configVersion");
+		propertiesToBeSkipped.add("submoduleCfg");
+		propertiesToBeSkipped.add("doGenerateSubmoduleConfigurations");
+		propertiesToBeSkipped.add("canRoam");
+		propertiesToBeSkipped.add("caseSensitive");
+		propertiesToBeSkipped.add("followSymlinks");
```

Comment:
Why do we skip this property? Is it from `archiveArtifacts`? This property could be set using Dynamic DSL. Silently ignoring properties that could be translated into DSL doesn't sound like a good idea 🤔

Reply:
Yeah this whole feature was added before I started working on this project. I can remove the entire feature if it doesn't make sense. 

I suspect the intention was to not give annoying error flags on properties that were not supported.